autogpt/prompt.py

```diff
@@ -82,6 +82,7 @@ def get_prompt() -> str:
         ),
         ("Execute Python File", "execute_python_file", {"file": "<file>"}),
         ("Generate Image", "generate_image", {"prompt": "<prompt>"}),
+        ("Convert Audio to text", "read_audio_from_file", {"image": "<file>"}),
```

Comment:
Why do you use “image”: instead of “file”: ?

Reply:
You are right. I changed it.